docs/DeveloperGuide.adoc

```diff
@@ -301,23 +301,47 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |picky eater |filter food preferences |cook food that matches my taste
 
-|`* * *` |beginner at cooking |easily search for recipes with the ingredients I already have |use up all the food in my fridge
+|`* * *` |frugal user |easily search for recipes with the ingredients I already have |use up all the food in my fridge
 
 |`* * *` |frequent user |save my favourite recipes |quickly navigate to them without having to search them up again
 
 |`* * *` |goal-oriented student |track my progress |see how far I came and how much further I have to go to reach my goal
 
-|`* * *` |user with allergies |exclude allegic ingredients | obtain recipes that are catered to me.
+|`* * *` |user with many recipes in the recipe book |filter recipes by ingredients |locate a recipe easily
 
-|`* * *` |demotivated user |choose a goal for myself | cook more meals and be motivated by my progress.
+|`* * *` |user with allergies |exclude allergic ingredients |obtain recipes that are catered to me
 
-|`* * *` |busy student |preselect meals for the week | save time deciding on what to eat each day.
+|`* * *` |user struggling to eat healthier |receive motivation for eating healthy meals |stay motivated on my goal
 
-|`**` |clueless user |ask for suggested recipes | chose a random recipe to start cooking.
+|`* * *` |user who cooks regularly |add my own recipes with the goals they fall under |progress in my goals when I cook my own unique meals
 
-|`*` |student on budget |choose recipes that require lower cost| save money
+|`* * *` |demotivated user |choose a goal for myself |cook more meals and be motivated by my progress
```

Comment:
I think the word "unmotivated" is a more accurate word to use here compared to "demotivated"!

Reply:
thanks for reviewing! ill make the changes :)